sale_variant_configurator/models/sale_order.py

```diff
@@ -168,7 +175,7 @@ def _onchange_product_attribute_ids_configurator(self):
     @api.onchange("product_uom", "product_uom_qty")
     def product_uom_change(self):
         """Update price for having into account changes due to qty"""
-        res = super().product_uom_change()
-        if not self.product_id:
+        res = {}
```

Comment:
Have you made sure that this method no longer exists to remove the call to the parent method?

Reply:
Yes, no longer exists on v16

Reply:
Then maybe you should rename it
https://github.com/OCA/sale-workflow/blob/16.0/sale_blanket_order/models/sale_orders.py#L115

Maybe this could have a conflict in some cases?

Reply:
Now you have to override the computed methods instead.

Reply:
Ready for review